source/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,58 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+
+Use skip to omit documents from the beginning of the list of returned
```

Comment:
It could be helpful to the user to use an inline literal for the `skip` function here and any mention of skip or sort following this.

Reply:
Also, I would add a description of the parameter that it takes in this sentence. E.g.: "Use skip to omit a specified number of documents from the beginning of the list of returned..."

Reply:
I don't do this for limit, sort, project, or sort right now -- I think I'd only make this change if we stayed consistent across pages, unless there is a reason for us to differ for some of these. I do think an inline literal makes sense here, so I'm going to make the change across pages and just revert it if we feel it isn't necessary.